libs/core/src/components/pds-tabs/pds-tab/pds-tab.scss

```diff
@@ -30,35 +38,35 @@ pds-tab {
 @mixin pds-button-style-reset {
   appearance: none;
   background-color: transparent;
-  border: 0;
+  border: var(--border-radius-none);
```

Comment:
Should a radius token be used for the border? Not a show-stopper.

Reply:
This is an oversight lol. Updated this as well. Good catch!